synfig-studio/src/gui/instance.cpp

```diff
@@ -359,10 +260,13 @@ studio::Instance::run_plugin(std::string plugin_path, bool modify_canvas, std::v
 	// Generate temporary file name
 	String filename_original = get_canvas()->get_file_name();
 	String filename_processed;
+	String filename_prefix;
+	if ( !is_absolute_path(filename_original) )
+		filename_prefix = temporary_filesystem->get_temporary_directory() + ETL_DIRECTORY_SEPARATOR;
```

Comment:
Someone can check if this temporary directory works fine on Windows?
If it points to a regular (=not admin) user has write access on the returned folder?
It could not be inside the folder Synfig is installed, but a system-defined one.
@mbasaglia @ice0

Reply:
on my system it doesn't even use /tmp, it uses ~/.sinfig/tmp

Reply:
My worry is on MS Windows :(

I had a bad experience on using it installed on a Windows PC  by IT crew. I couldn't save the file :O

Reply:
I think the filesystem gets constructed using `App::get_temporary_directory()` which is defined like that:
```c++
App::get_temporary_directory()
{
	return synfigapp::Main::get_user_app_directory() + ETL_DIRECTORY_SEPARATOR + "tmp";
}
```

Reply:
by the way, `temporary_filesystem->get_temporary_directory()` was already used in this function

Reply:
I don't think it's a reason to not merge this commit.
It just sounds a bell on my rings :)

Reply:
No worries, this dir is in `$HOME/Synfig` on Windows. It is proven to work as it is used for running plugins and autosave functionality. ^__^

Reply:
Well, I did had problems with 1.3.11
https://github.com/synfig/synfig/issues/1052

Actually, that's why I started to contributed to Synfig... tried to cleanup etl... in order to understand code. But I'm not a regular Windows user and building is too slow for Windows. So, I started to do other things... lol
Someday I gave up and opened the issue